update studio v3.3.2 (#1385)
vesoft-inc/nebula-docs: docs-2.0/nebula-studio/about-studio/st-ug-release-note.md

# Change Log

## v3.3.2(2022.05.19)

- Fix

>   - Fix the bug of rendering error when path has only starting vertex.

Reviewer: ```suggestion
  - Fix the rendering bug when a path has only one source vertex.
```
